bundles/org.openhab.core.model.script/src/org/openhab/core/model/script/actions/Audio.java

```diff
@@ -47,17 +47,23 @@ public static void playSound(@ParamDoc(name = "filename", text = "the filename w
 
     @ActionDoc(text = "plays a sound with the given volume from the sounds folder to the default sink")
     public static void playSound(@ParamDoc(name = "filename", text = "the filename with extension") String filename,
-            @ParamDoc(name = "volume", text = "the volume to be used") PercentType volume) {
+                                 @ParamDoc(name = "volume", text = "the volume to be used") PercentType volume) {
```

Comment:
What happened to the formatting here?

Reply:
I aligned all param docs for better readability, I can revert if you want.

Reply:
no, fine then.